crates/sncast/tests/helpers/devnet.rs

```diff
@@ -66,9 +70,48 @@ fn start_devnet() {
         "/constructor_with_params/target/dev/constructor_with_params_ConstructorWithParams",
         "CAST_WITH_CONSTRUCTOR",
     ));
+
+    rt.block_on(deploy_keystore_account());
+
     dotenv::from_filename(DEVNET_ENV_FILE).unwrap();
 }
 
+async fn deploy_keystore_account() {
+    let keystore_path = "tests/data/keystore/deployed_key.json";
+    let account_path = "tests/data/keystore/deployed_account_copy.json";
+
+    fs::copy("tests/data/keystore/deployed_account.json", account_path).unwrap();
+    env::set_var(KEYSTORE_PASSWORD_ENV_VAR, "123");
+
+    let address = get_address_from_keystore(keystore_path, account_path, KEYSTORE_PASSWORD_ENV_VAR);
+
+    mint_token(
+        &convert_to_hex(&address.to_string()),
+        9_999_999_999_999_999_999,
+    )
+    .await;
+
+    let args = vec![
+        "--url",
+        URL,
+        "--keystore",
+        keystore_path,
+        "--account",
+        account_path,
+        "account",
+        "deploy",
+        "--max-fee",
+        "99999999999999999",
+        "--class-hash",
+        DEVNET_OZ_CLASS_HASH,
+    ];
+
+    Command::new(cargo_bin!("sncast"))
```

Comment:
I don't really like testing cast with `sncast` binary, we should probably avoid it.
We could just deploy an account using `OpenZeppelinAccountFactory` from the sdk, maybe we could even hold deployed account file in repo to not have to create it from scratch (which would require us to write additional logic just for tests). wdyt?

Reply:
IMO it's overkill! this method is as regular before test action. Probably we could execute it in this particular test. Using OpenZeppelinAccountFactory will duplicate code from cast. I fought to use existing method but I had problem with imports